comps/comps-foreman-plugins-rhel7.xml

```diff
@@ -133,6 +133,7 @@
       <!--
         Do not edit this section manually and use ./comps_doc.sh script
       -->
+      <packagereq type="default">rubygem-smart_proxy_m2</packagereq>
```

Comment:
Could you move this to the appropriate section where the other plugins also live?

Reply:
Sure thing, I didn't notice this got put in a strange spot.  I think this got generated by `add_gem_package.sh`, is that script the latest recommended way to generate these?

Reply:
It should be, but the comps adding isn't the greatest code and could use some love.